backend/src/Controller/Package/PackageController.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace App\Controller\Package;
+
+use App\AutoMapping;
+use App\Controller\BaseController;
+use App\Request\Package\PackageCreateRequest;
+use App\Request\Package\PackageUpdateStateRequest;
+use App\Service\Package\PackageService;
+use stdClass;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use OpenApi\Annotations as OA;
+use Nelmio\ApiDocBundle\Annotation\Security;
+
+/**
+ * Create and fetch packages.
+ * @Route("v1/package/")
+ */
+class PackageController extends BaseController
+{
+    private $autoMapping;
+    private $validator;
+    private $packageService;
+
+    public function __construct(SerializerInterface $serializer, AutoMapping $autoMapping, 
+    ValidatorInterface $validator, PackageService $packageService)
+    {
+        parent::__construct($serializer);
+        $this->autoMapping = $autoMapping;
+        $this->validator = $validator;
+        $this->packageService = $packageService;
+    }
+
+    /**
+     * admin:Create new package by admin
+     * @Route("package", name="createPackage", methods={"POST"})
+     * @IsGranted("ROLE_ADMIN")
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Package")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\RequestBody(
+     *      description="new package",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="string", property="name"),
+     *          @OA\Property(type="number", property="cost"),
+     *          @OA\Property(type="string", property="note"),
+     *          @OA\Property(type="integer", property="carCount"),
+     *          @OA\Property(type="string", property="city"),
+     *          @OA\Property(type="integer", property="orderCount"),
+     *          @OA\Property(type="integer", property="status"),
```

Comment:
Please update the type of the "status" field from integer to string.

Reply:
done